app/assets/stylesheets/components/team_members.scss

```diff
@@ -1,8 +1,8 @@
-.list-item-padding {
+.membership-list.list-item-padding {
   margin-bottom: -5px;
 }
 
-.govuk-link {
+.membership-list.govuk-link--no-visited-state {
```

Comment:
since this is SCSS, you could just scope the top class and do your things in there:

```scss
.membership-list {
  .list-item-padding {
    margin-bottom: -5px;
  }

  .govuk-link {
    display: block;
  }
}
```

I've reverted back to styling the `govuk-link` as `--no-visisted-state` will just inherit from it. I believe.

Reply:
From my testing, it doesn't seem to work as `govuk-link`; I have to declare it as `govuk-link--no-visited-state` in the scss.  If you can get it to inherit, please let me know how!

Reply:
As per Slack discussion: we've agreed that we need styling on both `govuk-link` and `govuk-link--no-visited-state`.